### ISSUE TITLE
Unable to use global config node_modules for babel-presets

### ISSUE THREAD

**Issue by andrew-w-ross** [wallabyjs/quokka#89]
### Unable to use global config node_modules for babel-presets

I installed what was necessary for the global config in `~/.quokka` with `npm i babel-core babel-preset-env jsdom-quokka-plugin`

### Quokka.js file

```javascript
{
	"pro": true,
	"babel": {
		"env": "dev",
		"presets": ["env"]
	},
	"plugins": [
		"jsdom-quokka-plugin"
	]
}
```

### Sample repository link

Just open vscode in a new folder that is not in the global config and start quokka.

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v8.4.0, babel: v6.26.0)​​​​
​​Failed to run compilers on quokka.js, Error: Couldn't find preset "env" relative to directory "."​​
​​    at /home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19​​
​​    at Array.map (<anonymous>)​​
​​    at OptionManager.resolvePresets (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)​​
​​    at OptionManager.mergePresets (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)​​
​​    at OptionManager.mergeOptions (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)​​
​​    at OptionManager.init (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)​​
​​    at File.initOptions (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/index.js:212:65)​​
​​    at new File (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/file/index.js:135:24)​​
​​    at Pipeline.transform (/home/andrewr/.quokka/node_modules/babel-core/lib/transformation/pipeline.js:46:16)​​
​​    at _combinedTickCallback (internal/process/next_tick.js:131:7)​​
​​    at process._tickCallback (internal/process/next_tick.js:180:9)​​
​​    at _combinedTickCallback (internal/process/next_tick.js:131:7)​​
​​    at process._tickCallback (internal/process/next_tick.js:180:9)​​
```

### Code editor version

Visual Studio Code v1.15.1

### OS name and version

Linux : Ubuntu 17.04

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix is published in 1.0.58.

**Comment by andrew-w-ross** [comment]
@ArtemGovorov Yeah that fixed it, holy crap that was quick.
